sdk/inline_hooks.go

```diff
@@ -1,131 +0,0 @@
-package sdk
```

Comment:
I definitely _can_ split these back apart, but I'm not thrilled with the idea.  It will mean duplicating every type and function in this file with the only difference being replacing the word 'inline' with 'event'.  Even worse, the `AuthScheme`, `Channel`, `HookConfig`, and `Header` types will be completely identical.  That means either re-naming the existing ones (to something like `InlineHookAuthScheme`) and touching the inline hook resource (which I was trying to avoid) or leaving them named inconsistently.

Even beyond that, I'm not sure I agree that inline and event hooks are really fundamentally different things.  They're both just callbacks and only really differ in whether they're called synchronously or asynchronously (and the types of events you can get called back for).

Again, I definitely will do so, if you and @quantumew think it's the right way to go.  Just wanted to give my reasoning here.